app/src/main/res/layout/question_player_fragment.xml

```diff
@@ -116,5 +116,10 @@
         android:textSize="20sp"
         android:visibility="invisible" />
     </FrameLayout>
+    <View
```

Comment:
One blank line above this.

Also, to your recyclerview add `clipToPadding as false` otherwise your UI will get cut while scrolling:
<img width="473" alt="Screenshot 2020-04-03 at 12 13 40 PM" src="https://user-images.githubusercontent.com/9396084/78331878-8eda9f00-75a4-11ea-8fd6-dbd084d3a22a.png">

It was done in landscape mode just to show this scenario but it can happen in portrait mode also if the content is large enough.

Reply:
padding is removed